scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python3
+import glob
+import os
+import argparse
+from git import Actor
+import shutil
+import git
+from git import Repo
+import json
+import wget
+import sys
+
+def parse_ver_json(commit_name):
+    # download versions.json and find commit name
+    try:
+        ver_url = 'https://testing.zephyrproject.org/daily_tests/versions.json'
+        ver_file = wget.download(ver_url, 'versions.json')
+    except:
+        print("\nERROR: Can't download versions.json file. Script execution ended.")
+        sys.exit(1)
+
+    with open(ver_file) as json_file:
+            ver_data = json.load(json_file)
+
+            # Run for from the latest member of the list, mostlikely our commit will be at the end
+            for ver_name in reversed(ver_data):
+                try:
+                    ver_commit = ver_name.split("-")[-1]
+                    ver_commit = ver_name.split("g")[1]
+                except:
+                    print("\nCan't split that version name, skip it", ver_name)
+
+                if ver_commit == commit_name:
+                    print("\nFound existing version", ver_commit)
+                    os.remove("versions.json")
+                    return ver_name
+
+
+if __name__ == '__main__':
+
+        default_branch = 'master'
+        ver_folder_name = None
+
+        # Step 1. Setup variables below and add your credentials
+        upstream_repo = 'https://github.com/zephyrproject-rtos/test_results.git'
+        origin_repo = 'https://github.com/maksimmasalski/test_results.git'
+        branch_git_title = 'intel_daily_master_'
+        author_name = 'Maksim Masalski'
+        author_email = '<maksim.masalski@intel.com>'
+        company_name = 'Intel'
```

Comment:
I understand, read input, and enter all values.

Reply:
maybe read all these from a file would be better, so that the user name and password will not display in any log,
and use .gitignore to avoid this token file to be commited.  so every one can have their local setting.

```
		if iargparser.token == None:
			logging.info('load security token from local')
			with open(os.path.join(os.path.dirname(os.path.realpath(__file__)),"token.txt")) as file:
				iargparser.token = file.read()
```

Reply:
Git token you saved on your computer by yourself. So for example my git setup no need to enter password when push results to my repo on Github.